src/main/java/com/parasoft/demoapp/config/activemq/ActiveMQConfig.java

```diff
@@ -113,4 +113,11 @@ public static synchronized void resetInventoryActiveMqQueues() {
         inventoryRequestActiveMqQueue = new ActiveMQQueue(DEFAULT_QUEUE_INVENTORY_REQUEST);
         inventoryResponseActiveMqQueue = new ActiveMQQueue(DEFAULT_QUEUE_INVENTORY_RESPONSE);
     }
+
+    public static synchronized void resetInventoryRelatedQueues() {
```

Comment:
This method is duplicated with resetInventoryActiveMqQueues(). Keep resetInventoryActiveMqQueues() since it has unit test already.